mne/decoding/time_delaying_ridge.py

```diff
@@ -185,32 +267,40 @@ def fit(self, X, y):
 
         Parameters
         ----------
-        X : array, shape (n_samples, n_features)
+        X : array, shape (n_samples[, n_epochs], n_features)
```

Comment:
there's a docstring discrepancy between here and `_delay_time_series`...that OK?

Reply:
(referring to `[n_features]`)